.github/actions/build-okd/action.yaml

```diff
@@ -97,6 +104,70 @@ runs:
         make run-healthy
         make stop
 
+    - name: Push OKD images to production registry
+      if: success()
+      shell: bash
+      run: |
+        set -euo pipefail
+
+        cd ${GITHUB_WORKSPACE}/
+        # Only push to production if all tests passed
+        # This ensures we don't publish broken OKD images to production
+        TARGET_REGISTRY="${{ inputs.target-registry }}" ./src/okd/build_images.sh \
+          production \
+          "${{ inputs.okd-version-tag }}" \
+          "${{ inputs.ushift-gitref }}" \
+          "${{ inputs.target-arch }}"
+
+    - name: Cleanup staging registry
+      if: always()
+      shell: bash
+      continue-on-error: true
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+      run: |
+        set -euo pipefail
+
+        # GitHub Container Registry cleanup using gh CLI
+        # Deletes known staging packages
+        echo "Cleaning up staging packages..."
+
+        OWNER="${{ github.repository_owner }}"
+        VERSION="${{ inputs.okd-version-tag }}"
+
+        # List of staging packages to delete
+        packages=(
+          "okd-staging/cli"
+          "okd-staging/coredns"
+          "okd-staging/csi-snapshot-controller"
+          "okd-staging/haproxy-router"
+          "okd-staging/haproxy-router-base"
+          "okd-staging/kube-proxy"
+          "okd-staging/kube-rbac-proxy"
+          "okd-staging/operator-lifecycle-manager"
+          "okd-staging/operator-registry"
+          "okd-staging/pod"
+          "okd-staging/scos-${VERSION}"
+          "okd-staging/service-ca-operator"
+          "okd-staging/okd-release-arm64"
+        )
```

Comment:
We cannot duplicate this "knowledge" in the workflow. Can we add an option to the script to returh this list?